packages/atomic/cypress/e2e/smart-snippet-assertions.ts

```diff
@@ -4,84 +4,79 @@ import {should} from './common-assertions';
 import {SmartSnippetSelectors} from './smart-snippet-selectors';
 
 export function assertShowMore(display: boolean) {
-  it(`${should(display)} display the show more button`, () => {
-    SmartSnippetSelectors.showMoreButton().should(
-      display ? 'be.visible' : 'not.exist'
-    );
-  });
+  SmartSnippetSelectors.showMoreButton().should(
+    display ? 'be.visible' : 'not.exist'
+  );
 }
 
 export function assertShowLess(display: boolean) {
-  it(`${should(display)} display the show less button`, () => {
-    SmartSnippetSelectors.showLessButton().should(
-      display ? 'be.visible' : 'not.exist'
-    );
-  });
+  SmartSnippetSelectors.showLessButton().should(
+    display ? 'be.visible' : 'not.exist'
+  );
 }
 
 export function assertAnswerHeight(expectedHeight: number) {
-  it(`the answer should have a displayed height of ${expectedHeight}px`, () => {
-    SmartSnippetSelectors.truncatedAnswer()
-      .invoke('height')
-      .should('equal', expectedHeight);
-  });
+  SmartSnippetSelectors.truncatedAnswer()
+    .invoke('height')
+    .then((height) => {
+      const roundedHeight = Math.round(height!);
+      cy.wrap(roundedHeight).should('equal', expectedHeight);
+    });
 }
 
 export function assertCollapseWrapperHeight(expectedHeight: number) {
-  it(`the collapse wrapper should have a displayed height of ${expectedHeight}px`, () => {
-    SmartSnippetSelectors.collapseWrapper()
-      .invoke('height')
-      .should('equal', expectedHeight);
-  });
+  SmartSnippetSelectors.collapseWrapper()
+    .invoke('height')
+    .should('equal', expectedHeight);
 }
```

Comment:
Replace should('equal') by should('closeTo', expectedHeight, someDelta)
This will make more sense IMO.
We can go with a delta of .1 IMO

Reply:
nice find.